transformers/struct.go

```diff
@@ -239,6 +239,9 @@ func (t *structTransformer) addColumnFromField(field reflect.StructField, parent
 			return fmt.Errorf("failed to transform type for field %s: %w", field.Name, err)
 		}
 	}
+	if columnType == nil {
+		return nil // ignored
```

Comment:
This ensures backwards compatibility with the current situation where we ignore `reflect.Interface` early in `isTypeIgnored`.